docs/pages/installation/mesheryctl.md

```diff
@@ -62,6 +85,44 @@ Removing: /Users/lee/Library/Caches/Homebrew/mesheryctl--0.3.2.zip... (3.9MB)
 ==> No dependents found!
 ```
 
+## Kubernetes 
+
+**Install**
+
+<h6>Using mesheryctl</h6>
+
+<pre class="codeblock-pre">
+<div class="codeblock">
+ <div class="clipboardjs">
+ mesheryctl system context create k8s -p kubernetes -s
+ mesheryctl system start
+ </div></div>
+</pre>
+Don't have `mesheryctl`? Install with Bash, Brew, or Scoop.
+
+<h6>Using Helm</h6>
+
+<pre class="codeblock-pre">
+<div class="codeblock">
+ <div class="clipboardjs">
+ kubectl create ns meshery
+ helm repo add meshery https://meshery.io/charts/
+ helm install meshery meshery/meshery -n meshery
+ </div></div>
+</pre>
+Not a Helm user? Use the Meshery manifests directly.
+
+<h6>Using Manifests</h6>
+
+<pre class="codeblock-pre">
+<div class="codeblock">
+ <div class="clipboardjs">
+ $ git clone https://github.com/layer5io/meshery.git; cd meshery
+ $ kubectl create ns meshery
+ $ kubectl -n meshery apply -f install/deployment_yamls/k8s
```

Comment:
Will these "$"s be captured into the user's clipboard? If so, they'll need removed.

Reply:
No the "$"s don't get captured in the user's clipboard.

Reply:
Cool. In this case, let's be consistent with their use and incorporate the "$"s in the clipboardjs blocks above.